packages/orbit-components/src/Timeline/components/TimelineStep/index.js

```diff
@@ -0,0 +1,158 @@
+
+  return theme.orbit.paletteCloudNormalHover;
+};
+
+const getLineWidth = (desktop, isLeft, last, theme) => {
+  if (desktop) {
+    if (!isLeft) return css`calc(50% + ${theme.orbit.spaceSmall})`;
+    return "50%";
+  }
+
+  return "2px";
+};
+
+const ProgressLineStyled = styled.span`
+  ${({ desktop, theme, status, isLeft, last }) => css`
+    position: absolute;
+    width: ${getLineWidth(desktop, isLeft, last, theme)};
+    ${isLeft ? `left: 0` : `right: -20px`};
+    z-index: -1;
+    top: 40px;
+    ${!desktop && `left: 11px`};
+    background: ${renderStatus(status, theme)};
+    ${!desktop && `top: 0`};
+    height: ${desktop ? `2px` : `calc(100% + 20px)`};
+  `}
+`;
+
+ProgressLineStyled.defaultProps = {
+  theme: defaultTheme,
+};
+
+const TimeWrapperStyled = styled.div`
+  visibility: ${({ visible }) => (visible ? `visible` : `hidden`)};
+`;
+
+const IndentStyled = styled.div`
+  padding-left: ${({ theme }) => theme.orbit.spaceXSmall};
+`;
+
+IndentStyled.defaultProps = {
+  theme: defaultTheme,
+};
+
+const RelativeStyled = styled.div`
+  position: relative;
+`;
+
+const TimelineStep = ({ children, step, time, status }: Props) => {
+  const { state, dispatch } = useStatuses();
+  const { index, last } = useStep();
+
+  const getStatus = state.statuses[index + 1];
+
+  React.useEffect(() => {
+    dispatch({ type: "ADD_STATUS", payload: status || null });
+  }, [dispatch, status]);
+
+  return (
+    <>
+      <Hide on={["desktop", "largeDesktop"]}>
+        <RelativeStyled>
+          <Stack inline spacing="compact" spaceAfter="large">
+            {!last && <ProgressLineStyled status={getStatus} data-test="progressLine" />}
+            <IconWrapperStyled>
+              <StatusIcon status={status} />
+            </IconWrapperStyled>
+            <Stack flex direction="column">
+              <Stack inline spacing="condensed" align="center">
+                <Badge type={status}>
+                  <Text type={status || "secondary"}>{step}</Text>
+                </Badge>
+                <Text size="small">{time}</Text>
+              </Stack>
+              <IndentStyled>
+                <Text type={status ? "primary" : "secondary"}>{children}</Text>
+              </IndentStyled>
+            </Stack>
+          </Stack>
+        </RelativeStyled>
+      </Hide>
+      <Hide on={["smallMobile", "mediumMobile", "largeMobile", "tablet"]}>
+        <RelativeStyled>
+          <Stack inline direction="column" align="center" spacing="compact" spaceAfter="large">
+            <TimeWrapperStyled visible={!!status}>
+              <Text size="small">{time}</Text>
+            </TimeWrapperStyled>
+            <ProgressLineStyled data-test="progressLine" desktop isLeft status={status} />
+            <IconWrapperStyled>
+              <StatusIcon status={status} />
+            </IconWrapperStyled>
+            <ProgressLineStyled
+              data-testid="progressLine"
```

Comment:
We configured Testing Library to look for `data-test` instead of the default `data-testid` because that's the convention we were already using in Orbit.
```suggestion
              data-test="progressLine"
```

Reply:
You could notice, that I had `data-test` set already, it's just one forgotten. Thanks, that you notice it :)